### ISSUE TITLE
Why there are no descriptions?

### ISSUE THREAD

**Issue by agzam** [kilimchoi/engineering-blogs#1109]
Can descriptions be added? Otherwise, this collection of a bunch of URLs (albeit alphabetized) has little use. Maybe a script that goes through them and retrieves `document.title` would do?

**Comment by sysadmin-info** [comment]
> Can descriptions be added? Otherwise, this collection of a bunch of URLs (albeit alphabetized) has little use. Maybe a script that goes through them and retrieves `document.title` would do?

I agree. For technologies is fine as it is, for companies also, but for individuals categories at least one should be added. I do not know 100% of them and checking each site one by one is a nightmare. Maybe some chatgpt-based algorithm that can crawl these sites and summarize them? Hmm. I will think about it, but I am not sure is it possible.

**Comment by ahmedbo6211-glitch** [comment]
Good one
